src/project/qml/MuseScore/Project/internal/PropertyItem.qml

```diff
@@ -0,0 +1,110 @@
+
+RowLayout {
+    id: root
+
+    width: 100
+    spacing: 8
+
+    property int propertyNameWidth: -1
+    property int propertyValueWidth: 100
+    property NavigationPanel navigationPanel: null
+    property var propertyItem
+
+    signal changePositionOfIndex(int index)
+    signal propertyNameChanged(string newValue)
+    signal propertyValueChanged(string newValue)
+
+    StyledTextLabel {
+        Layout.preferredWidth: root.propertyNameWidth
+
+        text: root.propertyItem.propertyName ? root.propertyItem.propertyName : ""
+        font: ui.theme.bodyBoldFont
+        horizontalAlignment: Qt.AlignLeft
+        visible: root.propertyItem.isStandardProperty ? root.propertyItem.isStandardProperty : false
+    }
+
+    TextInputField {
+        Layout.preferredWidth: root.propertyNameWidth
+
+        currentText: root.propertyItem.propertyName ? root.propertyItem.propertyName : ""
+        visible: !root.propertyItem.isStandardProperty
+        hint: qsTrc("project", "Property")
+
+        navigation.name: root.propertyItem.propertyName + "PropertyName"
+        navigation.panel: root.navigationPanel
+        navigation.column: root.propertyItem.index * 3
+        navigation.onActiveChanged: {
+            if (navigation.active) {
+                root.changePositionOfIndex(index)
+            }
+        }
+
+        onCurrentTextEdited: function(newValue) {
+            root.propertyNameChanged(newValue)
+        }
+    }
+
+    TextInputField {
+        Layout.fillWidth: true
+
+        currentText: root.propertyItem.propertyValue ? root.propertyItem.propertyValue : ""
+        hint: root.propertyItem.isStandardProperty ? "" : qsTrc("project", "Value")
+        readOnly: root.propertyItem.isTopPanelProperty
+
+        navigation.name: root.propertyItem.propertyName + "PropertyValue"
+        navigation.panel: root.navigationPanel
+        navigation.column: root.propertyItem.index * 3 + 1
+        accessible.name: root.propertyItem.propertyName + " " + currentText
+        navigation.onActiveChanged: {
+            if (navigation.active && !root.propertyItem.isTopPanelProperty) {
+                root.changePositionOfIndex(index)
+            }
+        }
+
+        onCurrentTextEdited: function(newValue) {
+            root.propertyValueChanged(newValue)
+        }
+    }
+
+    FlatButton {
+        Layout.preferredWidth: root.propertyItem.isTopPanelProperty? 0 : 30
+        Layout.preferredHeight: root.propertyItem.isTopPanelProperty? 0 : 30
+
+        icon: IconCode.DELETE_TANK
+        opacity: !root.propertyItem.isStandardProperty
+        enabled: !root.propertyItem.isStandardProperty
+
+        navigation.name: root.propertyItem.propertyName + "Delete"
+        navigation.panel: root.navigationPanel
+        navigation.column: root.propertyItem.index * 3 + 2
+        accessible.name: "Delete"
+
+        onClicked: projectPropertiesModel.deleteProperty(root.propertyItem.index - 5) //5 to account for the 5 top panel properties
```

Comment:
This code is not reliable enough. It will stop working if someone adds new standard properties to the dialog